cypress/integration/plugins/observability-dashboards/3_trace_analytics_traces.spec.js

```diff
@@ -35,7 +35,7 @@ describe('Testing traces table', () => {
 
   it('Renders the traces table', () => {
     cy.contains(' (108)').should('exist');
-    cy.contains('03/25/2021 10:23:45').should('exist');
+    cy.contains('03/25/2021 17:23:45').should('exist');
```

Comment:
Had a similar issue with gantt-charts, can we remove the explicit "time equals" check and may be just check the date existence? 

Issue to gantt-charts: https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/493